test/e2e/pull_test.go

```diff
@@ -38,7 +38,8 @@ var _ = Describe("Podman pull", func() {
 
 		session := podmanTest.Podman([]string{"pull", "quay.io/libpod/ibetthisdoesntexist:there"})
 		session.WaitWithDefaultTimeout()
-		Expect(session).To(ExitWithError(125, "nitializing source docker://quay.io/libpod/ibetthisdoesntexist:there: reading manifest there in quay.io/libpod/ibetthisdoesntexist: unauthorized: access to the requested resource is not authorized"))
+		// FIXME: uncomfortable hardcoding of localhost:56789
+		Expect(session).To(ExitWithError(125, "nitializing source docker://quay.io/libpod/ibetthisdoesntexist:there: reading manifest there in localhost:56789/libpod/ibetthisdoesntexist: manifest unknown"))
```

Comment:
heads up, tests should still pass locally and I don't think we want to setup a local registry there right?
Thus I would think we need a regex or a `Or()` matcher to match both strings anyway

Reply:
This is thorny. As tests are currently written, e2e tests hard-force the use of `test/registries.conf`. No matter where they're run (CI, laptop, anywhere). This may need to be reevaluated, but I'm not bothering with any of that until I find out if this approach is viable.

Reply:
yes sure keep testing, just keep in mind that the end result must still work locally